stub-generator/matrix_analytics_stub_generator/kotlin.py

```diff
@@ -90,10 +90,10 @@ def compute_kotlin(schema: Schema) -> str:
 
     result += "\n"
     if not schema.members:
-        result += "    override fun getProperties(): Map<String, Any>? = null\n"
+        result += "    override fun getProperties(): Map<String, Any?> = emptyMap()\n"
```

Comment:
In this case it could stay as before, since the parent `fun getProperties()` accepts a nullable type.
Anyway, I let you judge.
(and I think we have 0 occurrences of this today)

Reply:
happy to revert this :+1: